tensorboard/backend/application.py

```diff
@@ -89,6 +89,8 @@
 PLUGINS_LISTING_ROUTE = "/plugins_listing"
 PLUGIN_ENTRY_ROUTE = "/plugin_entry.html"
 
+EXPERIMENTAL_PLUGINS_QUERY_PARAM = "expplugin"
```

Comment:
Could we call this `experimentalPlugin` instead? *exp* is ambiguous in
general, and especially so given that TensorBoard has a notion of
experiments that’s not what’s meant here.